styles/pages/Home.module.scss

```diff
@@ -419,37 +433,44 @@
     flex-direction: column;
     padding: 48px 16px 48px 16px;
     border-radius: 8px;
+    align-items: center;
+    align-content: center;
+
     color: $primary-black-1;
-    background-color: #f5f5f5;
+    background-color: #ffffff;
 
     .statRow {
-      width: 800px;
+      width: 1200px;
+      display: flex;
+      flex-direction: row;
+
     }
 
     .statColLeft {
       display: flex;
+      flex-direction: column;
       justify-content: center;
       align-items: center;
       float: left;
-      width: 50%;
+      width: 25%;
       .statIcon {
         color: black;
         height: 64px;
         width: 64px;
       }
       .statText {
         text-align: center;
-        padding-left: 15%;
+        //padding-left: 15%;
         width: auto;
       }
     }
-    .statColRight {
+    /*.statColRight {
```

Comment:
Can remove this commented section if not used

Reply:
Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1